libs/backend/src/cast_vote_records/build_cast_vote_record.ts

```diff
@@ -304,7 +300,52 @@ function buildCVRCandidateContest({
           },
         ],
       };
-    }),
+    });
+
+  // We include unmarked write-ins (write-ins without the bubble filled) as
+  // contest selections without an indication and with the allocation
+  // status unknown.
+  const unmarkedWriteInSelections: CVR.CVRContestSelection[] =
+    unmarkedWriteIns?.map((unmarkedWriteIn) => {
+      // We can only have unmarked write-ins on hand-marked ballots
+      assert(options.ballotMarkingMode === 'hand');
+      assert(options.image);
+
+      return {
+        '@type': 'CVR.CVRContestSelection',
+        ContestSelectionId: unmarkedWriteIn.optionId,
+        OptionPosition: contestOptionPositionMap
+          ? contestOptionPositionMap[unmarkedWriteIn.optionId]
+          : getOptionPosition({ contest, optionId: unmarkedWriteIn.optionId }),
+        Status: [CVR.ContestSelectionStatus.NeedsAdjudication],
+        SelectionPosition: [
+          {
+            '@type': 'CVR.SelectionPosition',
+            HasIndication: CVR.IndicationStatus.No,
+            NumberVotes: 1,
+            IsAllocable: CVR.AllocationStatus.Unknown,
+            Status: [CVR.PositionStatus.Other],
+            OtherStatus: 'unmarked-write-in',
+            CVRWriteIn: {
+              '@type': 'CVR.CVRWriteIn',
+              WriteInImage: buildCvrImageData(options.image),
+            },
+          },
+        ],
+      };
+    }) ?? [];
+
+  return {
+    '@type': 'CVR.CVRContest',
+    ContestId: contest.id,
+    Overvotes: vote.length > contest.seats ? contest.seats : 0, // VVSG 2.0 1.1.5-E.2
```

Comment:
How do overvotes and unmarked write-ins interact here? If a voter overvotes by adding an unmarked write-in, what will the CVR look like?

Reply:
In the NH case, we will reject the ballot at VxScan if it's an unmarked write-in overvote.

If there's a future were we don't reject those, and there's a possibility of adjudicating it as an overvote at VxAdmin, it would still appear as normal / undervote in the CVR. 

Because UWIs can only become votes through adjudication and the scanner CVRs are pre-adjudication, they should be treated as undervotes.

Reply:
Can't the voter choose to cast the ballot despite the overvote at VxScan?

I agree it makes sense to treat the unmarked write-in as an undervote in the pre-adjudication CVR.

Reply:
In the NH case they cannot choose to cast the ballot despite the overvote. @mattroe can correct me if I'm wrong.